R/Driver.R

```diff
@@ -37,48 +37,63 @@ setMethod(
     # TODO: Print more details
   })
 
-#' Connect to a ODBC compatible database
+#' Connect to an ODBC compliant database
 #'
-#' @inheritParams DBI::dbConnect
-#' @param dsn The Data Source Name.
-#' @param timezone The Server time zone. Useful if the database has an internal
-#' timezone that is _not_ 'UTC'. If the database is in your local timezone set
-#' to [Sys.timezone()]. See [OlsonNames()] for a complete list of available
-#' timezones on your system.
+#' This method is invoked when [DBI::dbConnect()] is called with the first argument
+#' [odbc::odbc()].
+#'
+#' @param drv An `OdbcDriver` object, likely outputted from [odbc::odbc()].
+#' @param dsn The data source name. For currently available options, see the
```

Comment:
Indent second and subsequent lines?

Reply:
[5afcce0](https://github.com/r-dbi/odbc/pull/647/commits/5afcce05f343840718fa13ef880b53741e295070).